config/default/field.field.node.agency_component.field_submission_api_url.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 41383913-49cf-4cac-9893-b525bd07ac4c
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_submission_api_url
+    - node.type.agency_component
+  module:
+    - link
+id: node.agency_component.field_submission_api_url
+field_name: field_submission_api_url
```

Comment:
What're your thoughts on retaining the machine name field_submission_api? If we proceed with changing it to field_submission_api_url, we need to be sure we're modifying references to the field that explicitly look for field_submission_api. I know jsonapi config has references to that, I'm surprised it didn't get included in your exported config. Also, the api code coming down the pipeline looks for it as well.